ocean_lib/models/v4/test/test_swap_fees.py

```diff
@@ -13,31 +13,13 @@
 from ocean_lib.models.v4.models_structures import ErcCreateData, PoolData
 from ocean_lib.models.v4.side_staking import SideStaking
 from ocean_lib.web3_internal.constants import ZERO_ADDRESS
-from tests.resources.helper_functions import get_address_of_type
+from tests.resources.helper_functions import deploy_erc721_erc20, get_address_of_type
 
 
 def deploy_erc721_token(config, web3, factory_deployer_wallet, manager_wallet):
```

Comment:
What I meant here is to use `deploy_erc721_erc20` instead of deploy_erc721_token because I modified `deploy_erc721_erc20` to return only erc721 token if the erc721 publisher wallet is provided. If both of the wallets are provided then that function returns erc721 and erc20. So, use `deploy_erc721_erc20` when needed, no need to keep `deploy_erc721_token`.

Reply:
I need to use that function for some assertions related to the event emitted when the ERC 721 token is created. Unfortunately, I'm not able to access the events using the helper function.

Reply:
Maybe we can update the helper function to assert the events as well and then use it where needed. Please let me know here and also @maldo-ctrl and @calina-c you can tell your opinion about it and after that, I can create a ticket with that.

Reply:
In my opinion I think that the helpers functions should be as simple as possible so that they do not need maintenance. Maybe if we start adding more and more things to them they stop being helper functions. If we decide to put the events in the helpers functions deploy erc721 and erc20, in that case, I would split up `deploy_erc721_erc20`  and I would make different functions, each one in charge of one thing and with the option to return the logs of a specific event.

Reply:
Ok, I agree. If there are no usages of `deploy_erc721_erc20` that we can use directly here, then no need to modify the helper function.